src/components/activeVotation/VotationResult.tsx

```diff
@@ -73,13 +75,13 @@ const VotationResult: React.FC<VotationResultProps> = ({ winners, backToVotation
       {showResultsTable &&
         (isStv ? <StvResultTable result={result} /> : <ResultsTable result={result} votationId={votationId} />)}
       <Divider m="3em 0" />
-      <HStack w="100%" justifyContent="space-between">
+      <WrapStack breakpoint={isThereAnyUpcomingVotations ? 650 : 450} w="100%" justifyContent="space-between">
```

Comment:
Ser at det andre plasser er brukt 479 og 690 som breakpoint, er det noen grunn for at disse er annerledes? Burde vi kanskje legge inn breakpointsene som variabler man kan bruke globalt?

Reply:
Dersom det ikke funker å behandle det individuelt er det fair, men hvis man kan finnes et felles breakpoint vil det føre til at ting blir mer konsistent, noe som vil være en fordel.